### PR TITLE
LTD-4176 Display sub status in quick summary tab.

### PR DESCRIPTION
**Pull request by stuaxo**
### Aim

Display the sub-status in the quick summary tab.

[LTD-4176](https://uktrade.atlassian.net/browse/LTD-4176)

[LTD-4176]: https://uktrade.atlassian.net/browse/LTD-4176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by saruniitr — APPROVED** [review]

File: unit_tests/caseworker/cases/views/test_quick_summary.py [hunk 1]
```diff
@@ -146,3 +147,35 @@ def test_case_summary_data(authorized_client, data_queue, data_standard_case):
...
+    value,
+    expected,
+):
+    # Based on test_case_details_sub_status
+    data_standard_case["case"]["data"]["sub_status"] = value
+    case_url = reverse("cases:case", kwargs={"queue_pk": data_queue["id"], "pk": data_standard_case["case"]["id"]})
+    response = authorized_client.get(case_url)
+
+    html = BeautifulSoup(response.content, "html.parser")
+    dt = html.find("dt", string=re.compile("Sub-status"))
```
Comment (kevincarrogan) [inline unit_tests/caseworker/cases/views/test_quick_summary.py:178]: This actually is just testing my change still. You’ve added a table header and you’re asserting a definition title exists so you’re actually finding the wrong thing here.

Comment (stuaxo) [reply unit_tests/caseworker/cases/views/test_quick_summary.py:178]: Doh!  That's where doing a change late afternoon gets me :D

File: pii-secret-exclude.txt [hunk 2]
```diff
@@ -110,3 +110,4 @@ unit_tests/caseworker/advice/views/test_trigger_list_view.py
 unit_tests/caseworker/advice/views/test_advice_view.py
 unit_tests/caseworker/advice/views/test_view_ogd_advice_countersign.py
 unit_tests/caseworker/queues/test_views.py
+unit_tests/caseworker/cases/views/test_quick_summary.py
```
Comment (kevincarrogan) [inline pii-secret-exclude.txt:113]: What required us to add this here?

In general we try not to add files to this and instead just ignore the specific line instead but I can't quite see what would have triggered this particular file in this diff.

Comment (stuaxo) [reply pii-secret-exclude.txt:113]: Nothing in this diff, but one of the other tests has the word "Email" in it's parameterisation (line 46), and this triggered it.

I guess the pre-commit saw the file had changed, and decided it didn't like the - kind of weird, as it's not in my change.

https://github.com/uktrade/lite-frontend/blob/bbda9d748fc7a2aaad776ccf5dc6818f09087496/unit_tests/caseworker/cases/views/test_quick_summary.py#L46

Comment (stuaxo) [reply pii-secret-exclude.txt:113]: Is ignoring the line, something like a `noqa` comment ?

Comment (stuaxo) [reply pii-secret-exclude.txt]: OK, removed it - and with `--amend` it didn't seem to register it this time so 🤷

Comment (kevincarrogan) [reply pii-secret-exclude.txt]: > Is ignoring the line, something like a `noqa` comment ?

`  # /PS-IGNORE` is what you're looking for.

**Review by kevincarrogan — APPROVED** [review]

**Pull request merged**